### PR TITLE
Introduce test-init to initialize test services

### PR DESCRIPTION
**Pull request by XavierGeerinck**
# Description

Introduce a `test-init.sh` file to initialize the services required to run the tests. This makes it easier to know what has to be started to run tests.

### CODE REVIEW

File: scripts/test-init.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+echo "Starting MQTT Broker"
+docker run -d --rm --name emqx -p 1883:1883 -p 8081:8081 -p 8083:8083 -p 8883:8883 -p 8084:8084 -p 18083:18083 emqx/emqx
+
+echo "Starting Rabbit MQ"
+docker run -d --rm --hostname my-rabbitmq --name my-rabbitmq \
+    -p 0.0.0.0:5672:5672 -p 0.0.0.0:15672:15672 \
+    rabbitmq:3-management
+
+echo "Starting MongoDB"
+docker run -d --rm --name mongodb -p 27017:27017 mongo
```
Comment (shubham1172) [inline scripts/test-init.sh:29]: Since we are adding two new components, were e2e tests for these components failing earlier?

Comment (XavierGeerinck) [reply scripts/test-init.sh:29]: These components are required to run the tests :) always have been, just not documented well

Comment (shubham1172) [reply scripts/test-init.sh:29]: Should we also add that to a readme - how to run e2e tests, as a separate item?

Comment (XavierGeerinck) [reply scripts/test-init.sh:29]: I was planning on including that in the new README

Comment (shubham1172) [reply scripts/test-init.sh:29]: Ok, it's already here in https://github.com/dapr/js-sdk/issues/236

**Review by shubham1172 — APPROVED** [review]

**Pull request merged**